wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -24,7 +25,7 @@
                 {% page_permissions page as page_perms %}
                 <tr {% if ordering == "ord" %}id="page_{{ page.id|unlocalize }}" data-page-title="{{ page.get_admin_display_title }}"{% endif %} class="{% if not page.live %}unpublished{% endif %} {% block page_row_classname %}{% endblock %}">
                     {% if show_ordering_column %}
-                        <td class="ord">{% if orderable and ordering == "ord" %}<div class="handle icon icon-grip text-replace">{% trans 'Drag' %}</div>{% endif %}</td>
+                        <td class="ord">{% if orderable and ordering == "ord" %}<div class="handle icon icon-grip text-replace" tabindex="0" data-orderer>{% trans 'Drag' %}</div>{% endif %}</td>
```

Comment:
let's change this to a button.

If you run into issues with styling you can use `all: initial;` or all unset
https://css-tricks.com/almanac/properties/a/all/

Reply:
Also let's call this `data-order-handle`